app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditViewModel.kt

```diff
@@ -26,6 +26,8 @@ class ProfileEditViewModel @Inject constructor(
   /** Specifies whether download access has been enabled by the user. */
   val isAllowedDownloadAccess: LiveData<Boolean> = isAllowedDownloadAccessMutableLiveData
 
+  val isProfileDeletionDialogShown = MutableLiveData<Boolean>()
```

Comment:
As per this https://stackoverflow.com/a/63409045

I suggest using `ObservableField` so that the changes are immediate.

Reply:
Yes but we are calling the showDeletionDialog on Presenter class. So should I introduce an interface to handle the property change?